[Merged by Bors] - Getting started guide (#452)
stackabletech/kafka-operator: docs/modules/getting_started/pages/first_steps.adoc

First, make sure that all the Pods in the StatefulSets are ready:

[source,bash]
----
kubectl get statefulset
----

The output should show all pods ready:

----
NAME                                 READY   AGE
simple-kafka-broker-default          3/3     5m
simple-zk-server-default             3/3     7m
----

Then, create a port-forward for the Kafka Broker:

----
include::example$code/getting-started.sh[tag=port-forwarding]
----

Create a file containing some data:

----
include::example$code/getting-started.sh[tag=kcat-create-data]
----

Write that data:

----
include::example$code/getting-started.sh[tag=kcat-write-data]
----

Read that data:

----
include::example$code/getting-started.sh[tag=kcat-read-data]
----

Check the content:

----
include::example$code/getting-started.sh[tag=kcat-check-data]
----

And clean up:

----
include::example$code/getting-started.sh[tag=kcat-cleanup-data]
----

You successfully created a Kafka cluster and produced and consumed data.

== What's next

> Have a look at the xref:ROOT:usage.adoc[] page to find out more about the features of the Operator such as encryption and authentication or how to connect to OPA.

Reviewer: ```suggestion
Have a look at the xref:ROOT:usage.adoc[] page to find out more about the features of the Kafka Operator.
```
